lazer/sdk/js/src/socket/resilient-websocket.ts

```diff
@@ -26,7 +29,9 @@ export class ResilientWebSocket {
   private maxRetryDelayMs: number;
   private logAfterRetryCount: number;
 
-  wsClient: undefined | WebSocket;
+  wsClient:
+    | undefined
+    | (Omit<WebSocket, "terminate"> & Partial<Pick<WebSocket, "terminate">>);
```

Comment:
Nice, might want to leave a comment explaining this type though since it won't be obvious to folks who don't know the typescript utility types as well

Reply:
done in d4cf06be69d3e5609a1c312cb621e99dd2e3c046